centreon/www/widgets/src/centreon-widget-topbottom/src/useLinkToResourceStatus.ts

```diff
@@ -0,0 +1,73 @@
+interface UseLinkToResourceStatus {
+  getResourcesStatusUrl: (data) => string;
+}
+
+const useLinkToResourceStatus = (): UseLinkToResourceStatus => {
+  const getDetailsPanelQueriers = (data): object => {
+    const uuid = data?.uuid;
+    const hostId = uuid?.split('-')[0]?.slice(1);
+    const serviceId = uuid?.split('-')[1]?.slice(1);
+
+    const resourcesDetailsEndpoint = `/centreon/api/latest/monitoring/resources/hosts/${hostId}/services/${serviceId}`;
```

Comment:
Same comment, use centreonBaseURL

Reply:
And you should move the file up to www/widgets/src so that it will be available for all upcoming widgets